tests/avif/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+LIST(APPEND TESTS_FILES
```

Comment:
look at `tests/jpeg/CMakeLists.txt` -- this will need `IF(AVIF_FOUND)` protection at least so we don't build the tests when avif isn't available